code/datums/emergency_calls/emergency_call.dm

```diff
@@ -100,7 +101,7 @@
 		to_chat(usr, "<span class='danger'>You are jobbanned from the emergency reponse team!</span>")
 		return
 
-	if(!istype(distress) || !SSticker.mode.waiting_for_candidates || distress.mob_max < 1)
+	if(!SSticker.mode.waiting_for_candidates || distress.mob_max < 1)
```

Comment:
Why this change if I may ask?

Reply:
I handled this the wrong way because i believed this was related to the gamemode code, but ultimately this checked that using `get_random_call()` returned a valid gamemode.

I changed this instead that my gamemode code sets this variable to someone thing valid so the check passes instead.